package.json

```diff
@@ -155,5 +155,9 @@
         "unzipper": "^0.10.10",
         "uuid": "^7.0.2",
         "webextension-polyfill-ts": "^0.13.0"
+    },
+    "resolutions": {
```

Comment:
nice!
Just for reference to other devs on the team: https://classic.yarnpkg.com/en/docs/selective-version-resolutions/ (I didn't know about `"resolutions"` until now)